vipe/graphviz/dot_wrapper.py

```diff
@@ -52,10 +52,9 @@ def add_node(self, name, node):
             name (string): name of the node
             node (vipe.pipeline.pipeline.Node): data about the node
         """
-        if name in self.get_reserved_node_names():
-            raise Exception('Node name "{}" not allowed since it is a '
-                            'reserved word.'.format(name))
-        
+        if self.__n_reg.contains(name):
```

Comment:
`___n_reg` stands for nodes registry, right? So the fix is narrowed down to checking whether name wasn't used before instead of comparing it to reserved words?